.storybook/polyfills.js

```diff
@@ -1,3 +1,19 @@
 import "core-js/es";
 
 import "zone.js/dist/zone";
+
+if (!Element.prototype.matches) {
```

Comment:
There's some funky formatting going on here ... does `core-js` ship a relevant polyfill? If so (it should?) we should prefer that over adding one ourselves